app/controllers/async_info_controller.rb

```diff
@@ -57,7 +57,8 @@ def user_data
         config_body_class: @user.config_body_class,
         feed_style: feed_style_preference,
         created_at: @user.created_at,
-        admin: @user.any_admin?
+        admin: @user.any_admin?,
+        apple_auth: @user.email.end_with?("@privaterelay.appleid.com")
```

Comment:
Previously, in the frontend we were checking `base_data` for a user's email with this ending. But we were never sending the email in the `base_data` 😅